drivers/volume/portworx/portworx.go

```diff
@@ -104,6 +105,10 @@ const (
 	pxServiceLocalEndpoint                    = "portworx-service.kube-system.svc.cluster.local"
 	mountGrepVolume                           = "mount | grep %s"
 	mountGrepFirstColumn                      = "mount | grep %s | awk '{print $1}'"
+	ddCopy                                    = "dd if=%s of=%s bs=${%s} count=1 seek=0"
```

Comment:
Can we use these commands as part of application spec using raw block ?

Reply:
Done.